app/views/main_template.scala.html

```diff
@@ -15,7 +15,6 @@
  *@
 
 @import config.AppConfig
-
```

Comment:
Did autoformat remove this line? It's better visually to distinguish the difference between the imports and the constructor